e2e/test/scenarios/dashboard-cards/dashboard-card-reproductions.cy.spec.js

```diff
@@ -1328,14 +1332,16 @@ describe("issue 31628", () => {
         scalarTitle.then($element => assertIsEllipsified($element[0]));
         scalarTitle.realHover();
 
-        popover().findByText(SMART_SCALAR_QUESTION.name).should("exist");
+        cy.findByRole("tooltip")
+          .findByText(SMART_SCALAR_QUESTION.name)
+          .should("exist");
 
         /**
          * it should show previous value tooltip on hover
          */
         cy.findByTestId("scalar-previous-value").realHover();
 
-        popover().within(() => {
+        cy.findByRole("tooltip").within(() => {
           cy.contains("34.72%").should("exist");
           cy.contains("• vs. previous month: 527").should("exist");
         });
```

Comment:
Similar to the [previous comment](https://github.com/metabase/metabase/pull/45030/files#r1663998698), this would read more naturally if we said something like:
```js
selector
  .should("contain", "foo")
  .and("contain", "bar");
  
```